features/networking/pod.feature

```diff
@@ -452,3 +452,56 @@ Feature: Pod related networking scenarios
     """
     OVN is functional on the cluster
     """
+  # @author anusaxen@redhat.com
+  # @case_id OCP-33413
+  @admin
+  @destructive
+  Scenario: xt_u32 kernel module functionality check from NET_ADMIN pods
+    Given I have a project
+    And I have a pod-for-ping in the project
+    Then evaluation of `pod.name` is stored in the :client_pod clipboard
+    Given I select a random node's host
+    #Making sure the iptables on latest centos are actually nf_tables as the xt_u32 is only supported with that
+    When I run commands on the host:
+      | iptables --version |
+    Then the step should succeed
+    Then the output should contain "nf_tables"
+
+    Given I obtain test data file "networking/centos_latest_admin.yaml"
+    When I run oc create as admin over "centos_latest_admin.yaml" replacing paths:
```

Comment:
here need `admin` user to create the server pod?

Reply:
This is a NET_ADMIN pod so need to be deal administratively

Reply:
it's ok just curious